### DISCUSSION TITLE
How to enable legend in PNG/SVG export option?

### DISCUSSION THREAD

**Discussion by MichielVervaart** [apexcharts/apexcharts.js#3696] [Q&A] [answered]
I have this code:
https://codepen.io/MichielVervaart/pen/oNPRXjm

I came up with this html2canvas solution, but i wish there was a better solution.
https://codepen.io/MichielVervaart/pen/abaEOOz

But when i click on the download PNG button it only exports the chart and not the legend. How can i also export the legend in the same picture?

**Comment by markgoedegebure** [comment]
This is a pretty good question, as a image of a graph doesn't say much without it's legend. 
I also see this is default behaviour for all chart types. There seems to be no reference of how to configurate this anywhere in the docs.

**Comment by feelou** [comment]
> I have this code:
> 
> https://codepen.io/MichielVervaart/pen/abaEOOz
> 
> But when i click on the download PNG button it only exports the chart and not the legend. How can i also export the legend in the same picture?

Hello, I tested your codepen link. And by exporting the visualization, we can clearly see the legend displayed on the exported image. How did you get there, please? Any particular option?

**Reply by MichielVervaart** [reply]
> > I have this code:
> > https://codepen.io/MichielVervaart/pen/abaEOOz
> > But when i click on the download PNG button it only exports the chart and not the legend. How can i also export the legend in the same picture?
> 
> Hello, I tested your codepen link. And by exporting the visualization, we can clearly see the legend displayed on the exported image. How did you get there, please? Any particular option?

Hey, 

For some reason i accidently changed the pen to my custom fixed version. I reverted it back to the right 'wrong' version and made a new pen with the solution in there. I used html2canvas to come up with the result.

**Reply by feelou** [reply]
Hello,thanks for your answer.
Yes I understand that one cannot directly with apexchart. I look at your solution. And thank you for sharing

**Reply by wastu01** [reply]
thanks

**Comment by MichielVervaart** [comment, marked as answer]
This seems to be fixed since both my pens started working. Too bad the devs didnt bother to mention it on this discussion.
